packages/valory/skills/twitter_scoring_abci/behaviours.py

```diff
@@ -103,7 +103,7 @@ def async_act(self) -> Generator:
             ).round_sequence.last_round_transition_timestamp.timestamp()
             # Reset the window if the window expired before checking
             self.openai_calls.reset(current_time=current_time)
-            if self.openai_calls.max_tweets_reached():
+            if self.openai_calls.max_calls_reache():
```

Comment:
That's not correct:

https://github.com/valory-xyz/IEKit/blob/65567c7dc4f146f0365d812ca40eeca5137f8026/packages/valory/skills/twitter_scoring_abci/models.py#L64